envs/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM python:3.10
-ARG BIOFORMATS2RAW="0.4.0"
+ARG BIOFORMATS2RAW="0.7.0"
```

Comment:
Could I ask to sneak in an additional version bump and make this [0.9.4](https://github.com/glencoesoftware/bioformats2raw/releases/tag/v0.9.4) ?

and also update [conda environment.yaml](https://github.com/haniffalab/webatlas-pipeline/blob/02738f42ba3b01e959fbd75d0e11be2d009f2abc/envs/environment.yaml#L6)